JEMS-CSE416/JEMS
# Add regions

**Updates**
- Added functionality to upload additional files to add on top of an existing map
- Updated map cards to have the appropriate image url

**Testing**

**How to review:**
```
git checkout main
git fetch
git checkout add_regions
cd client
npm start
```

[NY Map Files.zip](https://github.com/JEMS-CSE416/JEMS/files/13484437/NY.Map.Files.zip)
something like below: 

1. Go to browser, create a new map
<img width="1634" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/78514b4c-7a00-4fd9-b999-5f6b3408aa2c">

2. Add new region by clicking on the plus button on the left side panel
<img width="1633" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/03c83e95-7a3c-4686-83a2-03fdc6495b00">

3. Upload a file from NY Map Files
<img width="1638" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/460dff60-89f5-4ca7-9a64-4584657a2b74">

4. See added map regions + alert notifying success 
<img width="1631" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/751ac3b5-0425-4ccd-98ff-16fb9691e578">

Updated Map cards
<img width="1635" alt="image" src="https://github.com/JEMS-CSE416/JEMS/assets/53535722/b3bccc18-c3a1-453e-a18f-2200f518855f">
